### PR TITLE
Personal-portfolio : 

### PR DESCRIPTION
**Pull request by Tommymarvel**
## Personal portfolio mobile version skeleton.

In this personal portfolio mobile version skeleton:

- Added a toolbar and headline text.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  font-size: 20px;
+}
+
+nav {
+  display: none;
+}
+
+.Menu {
+  width: 35px;
+}
```
Comment (vechicin) [inline style.css:78]: Your hamburger button is a little too big. Kindly fix that as matching the template design is a project requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <nav>
+        <ul class="nav">
+          <li class="navItem"><a href="#">Portfolio</a></li>
+          <li class="navItem"><a href="#">About</a></li>
+          <li class="navItem"><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <img class="Menu" src="images/menu.png" alt="" />
+    </header>
+    <main id="intro_Section">
```
Comment (vechicin) [inline index.html:24]: According to the project requirements, this should be a `<section>` tag. Kindly fix this.

Screen Shot 2022-06-01 at 11 36 55

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Tommymarvel  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Good commit history
- [x] Correct GitHub Flow

## Required Changes ♻️

- [ ] You are missing a professional README. Kindly fix this as it is part of the [general requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#general-requirements). You should follow [this template](https://github.com/microverseinc/readme-template)

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Tommymarvel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  </head>
+  <body>
+    <header class="head">
+      <div class="background"></div>
+      <nav class="nav" >
+        <div class="logo">My Logo</div>
+      <div class="menu">
+      <img src="images/menu.png" alt="" >
+    </div>
+      </nav>
```
Comment (muneebulrehman) [inline index.html:18]: Hey, while fixing the previously raised issues, your navbar is broken. There is no top padding/margin as in the Figma template. Have a look at your application. Kindly fix it. Also, make the logo and hamburger icon evenly spaced. I would look much better that way. 🙂
image

File: index.html (hunk 2 above)
Comment (muneebulrehman) [reply index.html]: @Tommymarvel , kindly follow up with the required changes. Your navbar should be enclosed in a `header` tag and headline section should be wrapped in a `section` tag.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Tommymarvel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. You are almost there.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -87,15 +87,16 @@ nav {
...
 .nav {
   display: flex;
-  align-items: center;
   justify-content: space-between;
-  width: 100%;
-  margin-top: 0;
+  align-items: center;
+  /* width: 100%;
+  margin-top: 0; */
+  /* background-color: red; */
```
Comment (tresorsawasawa) [inline style.css]: # [OPTIONAL]
- [ ] Kindly remove all commented code to make you code clear and clean.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Tommymarvel 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**